eox_nelp/views.py

```diff
@@ -34,3 +40,26 @@ def info_view(request):
         json.dumps(response_data),
         content_type="application/json"
     )
+
+
+class NelpMFEConfigView(MFEConfigView):
+    """
+    Provides an API endpoint to get the MFE_CONFIG from site configuration with
+    nelp extra fields.
+    """
+
+    @method_decorator(cache_page(settings.MFE_CONFIG_API_CACHE_TIMEOUT))
```

Comment:
is this necessary, since you are calling super ?

Reply:
[2399c52](https://github.com/eduNEXT/eox-nelp/pull/17/commits/2399c52d60eebf2823e494f685b8be276f19da20)